libkbfs/tlf_edit_history.go

```diff
@@ -467,6 +472,7 @@ func (teh *TlfEditHistory) updateHistory(ctx context.Context,
 
 	// Which paths have been removed?
 	removed := make(map[string]bool)
+	removeNotifications := make(map[string]*keybase1.FSNotification)
```

Comment:
seems like `removeNotifications` now makes `removed` redundant. WDYT about removing that one?